pepmatch/tests/test_ouptput_files.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+"""
+Tests for the creation of file creation by Matcher objects
+"""
+from pandas import DataFrame as DF
+from os import remove
+from common_features import *
+from pepmatch import Matcher
+from pepmatch.matcher import VALID_OUTPUT_FORMATS
+
+@pytest.fixture
+def match(proteome_path, query_path):
+    return Matcher(
+            query=query_path,
+            proteome_file=proteome_path,
+            max_mismatches=0,
+            k=0,
+            output_format='csv'
+        )
+
+
+# fixtures and supporting
+@pytest.fixture
+def simple_dataframe():
+    return DF(data={"col1" : [1, 2], "col2": [3, 4]})
+
+def _creation_calls(_matcher, _path, _df):
+    if _path.exists():
+        remove(_path.__str__())
+
+    _matcher.output_name = _path
+    _matcher._output_matches(_df)
+
+
+# Actual tests
+def test_local_creation(match, simple_dataframe):
+    """Test creation given the local path"""
+    local_location = Path("local_result.csv")
+
+    _creation_calls(match, local_location, simple_dataframe)
+
+    assert local_location.exists()
+    remove(local_location.__str__())
+
+
+def test_relative_creation(match, simple_dataframe):
+    """Tests for creation given a relative path"""
+    relative_location = Path(f"../PEPMatch/relative.csv")
+
+    _creation_calls(match, relative_location, simple_dataframe)
+
+    assert relative_location.exists()
+    remove(relative_location.__str__())
+
+
+def test_absolute_creation(match, simple_dataframe):
+    """Tests creation given an absolute path"""
+    absolute_location = Path("absolute.csv").absolute()
```

Comment:
I think you'll need to actually create a full path like: `/full/path/to/file.csv`